framework/src/Volo.Abp.RabbitMQ/Volo/Abp/RabbitMQ/RabbitMqOptionalArguments.cs

```diff
@@ -0,0 +1,171 @@
+using System;
+using System.Collections.Generic;
+
+namespace Volo.Abp.RabbitMQ;
+
+public class OptionalArguments
```

Comment:
It contains the parameters of `queue` and `exchange`, which may be confusing.
I think this class is unnecessary.

Add two dictionary properties, and developers can add additional arguments to the `queue` or `exchange` according to their own needs.
This way is simpler and conforms to the design of RabbitMQ API.  what do you think?
```csharp
public IDictionary<string, object> QueueArguments { get; set; }

public IDictionary<string, object> ExchangeArguments { get; set; }
```